src/js/hca/components/veteran-information/BirthInformationSection.jsx

```diff
@@ -59,7 +59,7 @@ class BirthInformationSection extends React.Component {
               onValueChange={(update) => {this.props.onStateChange('cityOfBirth', update);}}/>
           <ErrorableSelect label="State"
               name="stateOfBirth"
-              options={states.USA}
+              options={states.USA.concat([{label: 'Other', value: 'Other'}]).sort((a, b) => a.label.localeCompare(b.label))}
```

Comment:
Can you put the output of this concat and sort in options-for-select.js instead? In here, we're going to end up running it a bunch of times, every time the component re-renders. Probably not noticeable on a desktop browser, but I've seen select components with lots of options cause some sluggishness on phones in one of our other forms.